doc/modules/multiclass.rst

```diff
@@ -14,45 +14,78 @@ Multiclass and multilabel algorithms
 
 The :mod:`sklearn.multiclass` module implements *meta-estimators* to solve
 ``multiclass`` and ``multilabel`` classification problems
-by decomposing such problems into binary classification problems. Multitarget
+by decomposing such problems into binary classification problems. ``multioutput``
 regression is also supported.
 
-- **Multiclass classification** means a classification task with more than
-  two classes; e.g., classify a set of images of fruits which may be oranges,
-  apples, or pears. Multiclass classification makes the assumption that each
-  sample is assigned to one and only one label: a fruit can be either an
-  apple or a pear but not both at the same time.
+- **Multiclass classification** produce single output that is categorical variable.
+  In other words, a single classification task with more than two classes.
 
-- **Multilabel classification** assigns to each sample a set of target
-  labels. This can be thought as predicting properties of a data-point
-  that are not mutually exclusive, such as topics that are relevant for a
-  document. A text might be about any of religion, politics, finance or
-  education at the same time or none of these.
+  *example:* classify a set of images of fruits which may be oranges, apples, or
+   pears. Multiclass classification makes the assumption that each sample is
+   assigned to one and only one label: a fruit can be either an apple or a pear
+   but not both at the same time.
 
-- **Multioutput regression** assigns each sample a set of target
-  values.  This can be thought of as predicting several properties
-  for each data-point, such as wind direction and magnitude at a
-  certain location.
+  - valid :term:`multiclass` representation for
+    :func:`~utils.multiclass.type_of_target` (`y`) are:
```

Comment:
This should be clickable and point to https://scikit-learn.org/stable/modules/generated/sklearn.utils.multiclass.type_of_target.html#sklearn.utils.multiclass.type_of_target